admin_frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Replace template names with proper product name. Create React App should be replaced with Arka. Please replace the logo with Arka logo.